### PR TITLE
feat(cli): register parameters from fixture files, accept from cli

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/test-runner/test/assets/parametrized/test.spec.ts [hunk 1]
```diff
@@ -15,8 +15,12 @@
  */
 
 import { fixtures as baseFixtures } from '../../..';
+import { registerParameter } from '../../../out/fixtures';
```
Comment (dgozman) [inline packages/test-runner/test/assets/parametrized/test.spec.ts:18]: Why do we import from 'out', and not use fixtures.registerParameter?

File: packages/test-runner/src/runner.ts [hunk 2]
```diff
@@ -62,17 +73,28 @@ export class Runner {
       hasSetup = hasSetup || fs.statSync(path.join(config.testDir, 'setup.ts')).isFile();
     } catch (e) {
     }
-    if (hasSetup)
+    if (hasSetup) {
+      global['setParameterValue'] = this.setParameterValue.bind(this);
+      global['setParameterValues'] = this.setParameterValues.bind(this);
+      global['before'] = (fn: Function) => this._beforeFunctions.push(fn);
+      global['after'] = (fn: Function) => this._afterFunctions.push(fn);
```
Comment (dgozman) [inline packages/test-runner/src/runner.ts:80]: nit: might as well remove from global after requiring setup.

**Review by dgozman — APPROVED** [review]

File: packages/test-runner/test/assets/parametrized/test.spec.ts (hunk 1 above)
Comment (pavelfeldman) [reply packages/test-runner/test/assets/parametrized/test.spec.ts:18]: i guess it is autocomplete

File: packages/test-runner/src/runner.ts (hunk 2 above)
Comment (pavelfeldman) [reply packages/test-runner/src/runner.ts:80]: sure, but we don't run any more code after it. we need a better solution to this anyways!

**Pull request merged**